### ISSUE TITLE
Emergency Fire detection

### ISSUE THREAD

**Issue by ADITYADAS1999** [World-of-ML/DL-Simplified#325]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : Emergency Fire detection</br>
:red_circle: **Aim** :  Having a fire detection system can significantly reduce damages and maximize fire control efforts. It is also one of the most fundamental steps we can take for fire safety measures. Even if we are sleeping or in a busy working, area</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/tharakan684/urecamain</br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name :  Aditya Das
- GitHub Profile Link :  https://github.com/ADITYADAS1999
- Email ID : adityamca123@gmail.com
- Participant ID (if applicable):
- Approach for this Project : 
  - We are going to use TensorFlow API Keras for building our model. Let’s first create our ImageDataGenerator for labeling our data.
  -  Load the dataset and used here for training.
  -  Then create train, test and valid sets from the dataset.
  -  Perform classification and plot the results using Matplotlib image plots.
- What is your participant role: SSOC 2023 Participant as a contributor

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by abhisheks008** [comment]
What are the deep learning methods you are going to use here for this project?
@ADITYADAS1999

**Comment by ADITYADAS1999** [comment]
> What are the deep learning methods you are going to use here for this project? @ADITYADAS1999


Here I am going to use two different deep learning methods that is CNN (convolutional neural network) that specifically used for image recognition and tasks that involve the processing of pixel data. And another model that I want to use that is inception as a deep neural network that consists of repeating components referred to as Inception modules.

**Comment by abhisheks008** [comment]
Great though process. Issue assigned to you @ADITYADAS1999
